### ISSUE TITLE
[MAS4.2.12] [Screen reader -Live Chat dialog] After selecting “Presentation mode” Narrator is not narrating the notification message information

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1749]
**Actual Result:**
After selecting “Presentation mode” Narrator is not narrating the notification message information present in the Live chat dialog.
When the user again press the ESC key focus in on live chat pane reading as document .
Same issue for split editor control also.

**Expected Result:**
Screen Reader should narrate the notification message after “Presentation mode”
Example: After selecting “Presentation mode narrator should narrate as Full screen presentation modes or Press the ESC key to exit full screen presentation mode.

**Note:** Issue is repro’ing with NVDA 
**Note:** Same issue repro for all dialog controls throughout the application 
Note: Same issue repro for below path
Navigate to the Open bot ->Select the Open bot control->Open bot dialog ->Connect the bot ->Restart conversation menu
1.When the user restart the conversation screen reader is not notifying any information.
Please refer the attachment:MAS4.2.12_Restart conversation.zip

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened
3.Navigate to the open bot and select it  
4. Dialog gest opened and connect the open bot  
5. Live chat screen gets opened 
6.Navigate to the Presentation mode control and select it 
7.Again press the esc key observe the issue.

**User Impact:**
Users who rely on screen readers will affect to navigate if Screen reader is not narrating the change notification message when we are selecting the presentation mode.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Browser: Edge
Application: Bot Framework V4 Emulator
Screen reader: Narrator 

**MAS Reference Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b19c05bd-d82f-4d6d-a8e5-7081bf5c2dba}](url)
[MAS4.2.12_Presentation.zip](https://github.com/microsoft/BotFramework-Emulator/files/3519136/MAS4.2.12_Presentation.zip)
[MAS4.2.12_Restart conversation.zip](https://github.com/microsoft/BotFramework-Emulator/files/3519218/MAS4.2.12_Restart.conversation.zip)

[MAS4.2.12_Presentation_NVDA.zip](https://github.com/microsoft/BotFramework-Emulator/files/3545944/MAS4.2.12_Presentation_NVDA.zip)

**Comment by corinagum** [comment]
Since Presentation Mode is currently not accessible and not up to spec, we will be instead HIDING presentation mode temporarily until we are ready to make improvements to the feature. I will close this issue and open a new one where the team and customers can discuss what features they are looking for in presentation mode.

**Comment by tonyanziano** [comment]
Presentation mode was disabled, so this bug is no longer relevant.

**Comment by Kumar2608** [comment]
As per developer comments We are closing this bug .
